test/integration_tests/test_create_executor.py

```diff
@@ -35,12 +38,10 @@ def test_create_executor_should_create_table_in_storage(self):
         plan_node = CreatePlan(dummy_table, columns, False)
 
         createExec = CreateExecutor(plan_node)
-        url = createExec.exec()
+        metadata = createExec.exec()
 
-        # test if we have a table created in our storage
-        df = load_dataframe(url)
-        self.assertEqual(2, len(df.columns))
-        self.assertEqual(df.columns, ['id', 'name'])
+        row_iter = StorageEngine.read(metadata)
+        self.assertFalse(any(True for _ in row_iter))
```

Comment:
I think this is still a work in progress. So we should drop this for now and maybe add with a later PR.

Reply:
Fixed by 6de611c